src/utils/__tests__/jwtparams.test.ts

```diff
@@ -0,0 +1,50 @@
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { JWTParams } from "../config";
+
+describe("Validate an unknown type as JWTParams Interface", () => {
+  it("should validate a correct JWT Param", () => {
+    const JWTParamMock: unknown = {
+      ENABLE_JWT: true,
+      JWT_TOKEN_ISSUER: "issuer" as NonEmptyString,
```

Comment:
All the casting to `NonEmptyString` in this file are useful, because the main object has type equals to `unknown`

Reply:
we put the main object as `unknown` because the definition of type JWTParam is a bit complex and we would test it using decode, so avoid error on editor or a at compile time.

Reply:
@BurnedMarshal resolved by https://github.com/pagopa/hub-spid-login-ms/pull/37/commits/58033e0c263b483fb1838883c3185744b768cb2a